projects/ng2-signalr/package.json

```diff
@@ -1,14 +1,16 @@
 {
-  "name": "ng2-signalr",
-  "version": "12.0.0",
+  "name": "@henkkelder/ng2-signalr",
```

Comment:
```suggestion
  "name": "ng2-signalr",
```

Reply:
This will only work if this is actually merged into this repos. 
Since my first change was from 2021 and nothing happened with my PR I needed a workable solution.